packages/rocketchat-federation/server/index.js

```diff
@@ -0,0 +1,17 @@
+// Federation export
+const Federation = {
```

Comment:
```suggestion
export const Federation = {
```

Reply:
this was actually a copy paste from another file that I forgot to "normalize", good catch!